woocommerce/woocommerce-ios
# Variations: Release attribute edit feature

Closes: #3521 

## Description

This PR releases the "edit attributes" feature for product variations. This is part of a larger project to support adding/editing variations for variable products.

The "edit attributes" feature currently consists of an "Attributes" section in the product variation details. In the Attributes section, you can view and change the selected options for each attribute in the variation.

<img src="https://user-images.githubusercontent.com/8658164/107508573-259ac380-6b99-11eb-8682-a32bc6914b37.gif" width="300px">


## Changes

* Removes the condition in `ProductVariationFormActionsFactory` that puts the edit attribute feature behind the `addProductVariations` feature flag.

## Testing

1. Make sure you have at least one variable product with variations on your store.
2. Go to the **Products** tab in the app.
3. Select a variable product.
4. Select **Variations**.
5. Select a product variation.
6. In the product variation details, confirm you see an **Attributes** section (below the Price settings) containing the currently selected attribute options for this variation.
7. Select **Attributes**.
8. Confirm you see a list of the attributes for this variable product (e.g. Color or Size) and the currently selected options for each attributes (e.g. Green or Medium).
9. Select one of the attributes.
10. Confirm you see a list of all the available options for that attribute, including an "Any Attribute" option at the top of the list.
11. Select a different option and go back.
12. On the Attributes screen, confirm you now see a **Done** button in the top right. Select Done.
13. Confirm you are taken back to the product variation detail screen with the newly selected attributes in the Attributes section, and an Update button in the top right. Select Update.
14. Confirm the variation is updated remotely on your store, and you now see the newly selected attributes throughout the app (variation details, variations list).
15. Repeat the steps above but _don't_ save your new attribute selections, and confirm the app behaves as expected (you can discard your changes and no changes are made remotely).

## Submitter Checklist

Update release notes:
